src/ansi-c/library/unistd.c

```diff
@@ -319,3 +319,39 @@ ret_type _read(int fildes, void *buf, size_type nbyte)
   __CPROVER_HIDE:;
   return read(fildes, buf, nbyte);
 }
+
+/* FUNCTION: sysconf */
+
+#ifndef __CPROVER_ERRNO_H_INCLUDED
+#  include <errno.h>
+#  define __CPROVER_ERRNO_H_INCLUDED
+#endif
+
+long __VERIFIER_nondet_long(void);
+__CPROVER_bool __VERIFIER_nondet___CPROVER_bool(void);
+
+long sysconf(int name);
+
+// This overapproximation is based on the sysconf specification available at
+// https://pubs.opengroup.org/onlinepubs/9699919799/functions/sysconf.html.
+long sysconf(int name)
+{
+__CPROVER_HIDE:;
+  (void)name;
+  long retval = __VERIFIER_nondet_long();
+  __CPROVER_assume(retval >= -1);
+
+  // Spec states "On error, -1 is returned and errno is set to indicate
+  // the error (for example, EINVAL, indicating that name is invalid).
```

Comment:
Maybe remove the "for example" (which was taken from the manpage for sysconf), because it makes it seem like there may be other possible values